public/pages/EditFeatures/containers/EditFeatures.tsx

```diff
@@ -243,11 +247,72 @@ export function EditFeatures(props: EditFeaturesProps) {
     }
   };
 
+  const renderAdvancedSettingsToggle = () => (
+    <EuiText
+      className="content-panel-subTitle"
+      onClick={()=>{setShowAdvancedSettings(!showAdvancedSettings);}}
+    >
+      <EuiLink>
+        {showAdvancedSettings ? 'Hide' : 'Show'}
+      </EuiLink>
+    </EuiText>
+  );
+
+  const renderAdvancedSettings = () => (
+    <Field
+      name="shingleSize"
+      validate={validatePositiveInteger}
+    >
+      {({ field, form }: FieldProps) => (
+        <EuiFormRow
+          label="Window size"
+          helpText={
+            <EuiText className="content-panel-subTitle">
+              Set the number of data points to consider in a detection
+              window. We recommend you choose this value based on your actual
+              data. If you expect missing values in your data or if you want
+              the anomalies based on the current data points, choose 1. If
+              your data is continuously ingested and you want the anomalies
+              based on multiple data points, choose a larger window size.
+              <EuiLink
+                href="https://opendistro.github.io/for-elasticsearch-docs/docs/ad/"
+                target="_blank"
+              >
+                Learn more <EuiIcon size="s" type="popout" />
+              </EuiLink>
+            </EuiText>
+          }
+          isInvalid={isInvalid(field.name, form)}
+          error={getError(field.name, form)}
+        >
+          <EuiFlexGroup gutterSize="s" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiFieldNumber
+                id="shingleSize"
+                placeholder="Window size"
+                data-test-subj="shingleSize"
+                {...field}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText>
+                <p className="minutes">points</p>
```

Comment:
Feel `points` not so easy for new user to understand. Consult tech writer about this. If we use`points`, suggest to add some description about what "points" means. One point means features' aggregation results for one detection interval.

Reply:
The UX designer confirmed that this was reviewed by a tech writer. The description below the config box provides more context to the user.

I can relay your feedback to the UX designer if you feel that it is still not clear with the description.

Reply:
> The UX designer confirmed that this was reviewed by a tech writer. The description below the config box provides more context to the user.

Cool, thanks. I'm ok if tech writer confirms.

Reply:
I synced up with the tech writer, and we decided to change "points" (and "data points" in the description) to "intervals", since this more accurately describes what the window size is referring to.

Reply:
I think "intervals" is better. Thanks.